mbq/client/__version__.py

```diff
@@ -4,4 +4,4 @@
 __license__ = 'Apache 2.0'
 __title__ = 'mbq.client'
 __url__ = 'https://github.com/managedbyq/mbq.client'
-__version__ = '1.4.3'
+__version__ = '1.4.4'
```

Comment:
Nit: new features bump the minor version, not patch. Doesn't actually matter since we've never been strict about semver.